deployment/scripts/uuid_migrate.sh

```diff
@@ -0,0 +1,114 @@
+#! /bin/bash
+
+set -ex
```

Comment:
`-u` is good, i introduced it in metrics with a refactor, but it requires a lot more careful variable use than what we have so far.